src/mappening/__init__.py

```diff
@@ -0,0 +1,26 @@
+from mappening.utils.secrets import APP_SECRET_KEY
+
+from flask import Flask
+from flask_cors import CORS, cross_origin
+
+from mappening.api.events import events
+from mappening.api.pages import pages
+from mappening.api.locations import locations
+# from mappening.api.users import users             # TODO: finish
+# from mappening.api.preferences import preferences # TODO: finish
+from mappening.api.utils.tokenize import tokenize
+from mappening.auth.auth import auth
+
+# Configure app and register blueprints
+app = Flask(__name__)
```

Comment:
I can understand importing the above things here to simplify importation. But why also setup app here rather than app.py. Similarly every importation of mappening setups up app, not very modular. Maybe make this a function called createapp then call that in app.py.

Reply:
This is from the tutorials for flask structures I found online and a lot of the github repos I looked at. e.g. https://damyanon.net/post/flask-series-structure/ and other repos. If you think that would be better feel free